### PR TITLE
Create infectious_disease_and_sequelae_by_agent.yaml

### PR DESCRIPTION
**Pull request by nicolevasilevsky**

### CODE REVIEW

File: src/patterns/dosdp-patterns/infectious_disease_and_sequelae_by_agent.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  vars:
+  - agent
+
+def:
+  text: A disease and associated sequelae caused by infection with %s.
+  vars:
+  - agent
+
+equivalentTo:
+  text: '''%s'' and ''disease arises from feature'' some %s'
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/infectious_disease_and_sequelae_by_agent.yaml:47]: the grouping class should be a union of

 - D
 - disease and 'disease arises from feature' some D

Comment (nicolevasilevsky) [reply src/patterns/dosdp-patterns/infectious_disease_and_sequelae_by_agent.yaml:47]: would COVID-19 and sequelae look like this?

image

Comment (nicolevasilevsky) [reply src/patterns/dosdp-patterns/infectious_disease_and_sequelae_by_agent.yaml:47]: or is it meant to be like this?
image

but this causes the class to be equiv to COVID-19

Comment (nicolevasilevsky) [reply src/patterns/dosdp-patterns/infectious_disease_and_sequelae_by_agent.yaml:47]: or is it meant to be an 'or'?
image

**Comment by nicolevasilevsky** [comment]
this will be revisited another time and we'll redo this.

**Pull request closed without merging**